packages/windows/download_appveyor_build.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+require "pathname"
+require "open-uri"
+require "veyor"
+require "octokit"
+
+def download(url, local_file)
+  ssl_verify_mode = nil
```

Comment:
Why is this needed?

Reply:
Thanks, there is no need to use it. removed.